### DISCUSSION TITLE
scaling of profile elements is silly at wider resolutions and lower UI scales

### DISCUSSION THREAD

**Discussion by tovc** [ppy/osu#25497] [Ideas]
The team supporter banners are far taller than most elements, the image banner is trimmed vertically, and the rank graph is just too long. This might all be intentional, but it's silly if it is. This screenshot is taken at 0.8x scale and 21.5:9
![osu_2023-11-18_11-09-31](https://github.com/ppy/osu/assets/49025619/56685f23-2e12-44b0-b023-0f44c0a1eaa5)

This all looks far more normal at 1.0x scale and 16:9. There is definitely an argument to be made that even this is a tad oversize
![osu_2023-11-18_11-15-35](https://github.com/ppy/osu/assets/49025619/0d753c5f-3bfd-45fe-8bf3-0761afb3e04b)

This is solved on the web by limiting the width of the profile, which looks far more reasonable
![image](https://github.com/ppy/osu/assets/49025619/e3c89af2-e121-4a27-810c-757c5847d445)

**Comment by peppy** [comment]
Definitely seems like there should be a maximum width spec for the overlay.

**Reply by tovc** [reply]
I find that the full width beatmap listing works fine
![osu_2023-11-18_14-16-02](https://github.com/ppy/osu/assets/49025619/b0575f04-9e3e-437c-9e9a-1398fbed4eb1)

Some screens like the ranking panel and news just look a bit silly with all the dead space
![osu_2023-11-18_14-17-33](https://github.com/ppy/osu/assets/49025619/7178545c-da75-43d7-b684-a6de28c59fda)
![osu_2023-11-18_14-19-50](https://github.com/ppy/osu/assets/49025619/a28e3f63-2f93-4889-90e5-85273d5b6539)

The chat window could also do with being centered; always needing to look over to the left isn't perfect
![osu_2023-11-18_14-19-31](https://github.com/ppy/osu/assets/49025619/b2271b30-e3ac-447d-97a2-bcb9fc7e8bd0)

Honestly, a lot of the UI seems like it's purely designed for 16:9 aspect ratios, with more or less everything being anchored to the edge of the screen
![osu_2023-11-18_14-21-52](https://github.com/ppy/osu/assets/49025619/cda096bc-13ba-484d-90bb-482d5b23c6f6)

A lot of this stuff can just be reduced to nitpicking, and solved by simply using screen scaling. For the most part, I'd agree with that sentiment, but the vast majority of mobile phones are wider than 16:9, so I'd expect this kind of thing to become a more common issue.
